src/style.css

```diff
@@ -0,0 +1,24 @@
+*,
+*:after,
+*:before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.star__wrapper {
+  display: flex;
+}
+
+.star-block {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star-block__active {
```

Comment:
don't add extra classes unless completely necessary. Try to use :nth-child and delete star-block__active class